src/screens/Signup/StepInfo/index.tsx

```diff
@@ -46,13 +48,43 @@ export function StepInfo({
 
   const inviteCodeValueRef = useRef<string>(state.inviteCode)
   const emailValueRef = useRef<string>(state.email)
+  const prevEmailValueRef = useRef<string>(state.email)
   const passwordValueRef = useRef<string>(state.password)
 
-  const onNextPress = React.useCallback(async () => {
+  const [hasWarnedEmail, setHasWarnedEmail] = React.useState<boolean>(false)
+
+  const tldtsRef = React.useRef<typeof tldts>()
+  React.useEffect(() => {
+    // @ts-expect-error - valid path
+    import('../../../../node_modules/tldts/dist/index.cjs.min.js').then(
```

Comment:
does `import('tldts/dist/index.cjs.min.js')` not work?

Reply:
will try again, but it did not. read in a SO that you actually have to supply the full path

Reply:
oh, okay without `node_modules` infront it does work. nice, ty